test-cases/performance/perf-regression-throughput-125gb-with-ebs-gp3-volume.yaml

```diff
@@ -0,0 +1,42 @@
+test_duration: 580
```

Comment:
Let's use a shared based yaml for these tests like the disk-and-cache we already have, but with added "configuration" file that overrides the parameters that we need (instance type + EBS disks").

Reply:
I Created new yamls and updated jenkins files with shared configs